### PR TITLE
Unfluent

### PR DESCRIPTION
**Pull request by LoganWright**

### CODE REVIEW

File: Tests/VaporTests/AuthTests.swift [hunk 1]
```diff
@@ -23,9 +25,9 @@ class AuthTests: XCTestCase {
         
         droplet.database = database
         
-        let authMiddleware = AuthMiddleware(user: User.self)
-        droplet.middleware.append(authMiddleware)
-        
+//        let authMiddleware = AuthMiddleware(user: Userz.self)
```
Comment (tanner0101) [inline Tests/VaporTests/AuthTests.swift]: This should get deleted.

File: Sources/Vapor/Droplet/Droplet+Run.swift [hunk 2]
```diff
@@ -89,4 +77,21 @@ extension Droplet {
             ]
         )
     }
+
+    /// adds a serve command if it hasn't been overridden
+    // FIXME: Should probably add this first off if possible in future so that users can override w/o 
+    // a check like this
+    private func addServeCommandIfNecessary(servers: [String: ServerConfig]? = nil) {
```
Comment (tanner0101) [inline Sources/Vapor/Droplet/Droplet+Run.swift]: is there any cleaner way to do this?

File: vapor.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "meta": {
+        "a config file for toolbox :)"
+    },
+    "flags": {
+        "build": {
+            "macos": [
+                "-DVAPOR_INCLUDED"
```
Comment (tanner0101) [inline vapor.json]: I think just `-DVAPOR` is good

File: Sources/Vapor/Droplet/Droplet+Run.swift (hunk 2 above)
Comment (LoganWright) [inline Sources/Vapor/Droplet/Droplet+Run.swift]: yes, just for record since we discussed elsewhere. This is going to be addressed in another PR 👍
